docs/01-overview/main-areas/telemetry/telemetry-03-traces.md

```diff
@@ -170,10 +188,74 @@ stringData:
   password: XXX
 ```
 
-### Step 5: Rotate the Secret
+### Step 6: Rotate the Secret
 
 As used in the previous step, a Secret referenced with the **secretKeyRef** construct can be rotated manually or automatically. For automatic rotation, update the Secret's actual values and keep the Secret's keys stable. TracePipeline watches the referenced Secrets and detects changes, so the Secret rotation takes immediate effect. When using a Secret owned by the [SAP BTP Operator](https://github.com/SAP/sap-btp-service-operator), you can configure `credentialsRotationPolicy` with a specific `rotationFrequency` to achieve an automated rotation.
 
+## Kyma Components with tracing capabilities
+
+Kyma bundles several modules which are potentially involved in user flows and with must propagate the trace context properly and optional provide custom spans.
```

Comment:
Is this sentence complete? Can you elaborate on its meaning?

Reply:
I improved it. I agree that a general explanation of the topic would help. That would need to be done in the intro. I might add it at a later time.